packages/website/docusaurusConfig.ts

```diff
@@ -158,6 +159,9 @@ const config: Config = {
   projectName: 'typescript-eslint',
   clientModules: [require.resolve('./src/clientModules.js')],
   presets: [['classic', presetClassicOptions]],
+  customFields: {
+    rules: rulesMeta,
+  },
```

Comment:
Whoa, TIL about `customFields`. Thank you! This would have made some work @bradzacher and I were doing recently for rule attribute generation a little easier, maybe. 

Alternately, is there a reason to use this over `import { rulesMeta } from '---/rulesMeta'`?

Reply:
Yes, you can see I only pull in the relevant information that we actually need on client side, otherwise we pull in message IDs, schema, etc., which we don't immediately need. This data is available as a global context, so better keep it small.